index.js

```diff
@@ -88,9 +88,7 @@ module.exports = {
     // - application has ember-cli-fastboot dependency.
     this._needsFastBootSupport = this._config.enabled &&
       this._config.delivery.includes('header') &&
-      // TODO: check if application has ember-cli-fastboot-dependency
-      // https://github.com/rwjblue/ember-cli-content-security-policy/issues/116
-      true;
+      new VersionChecker(this).for('ember-cli-fastboot').exists();
```

Comment:
Changing this should (at least in theory) help identify if the _project_ has `ember-cli-fastboot` installed, and may help the linking scenario to allow us to unskip it.

```suggestion
      new VersionChecker(this.project).for('ember-cli-fastboot').exists();
```

Reply:
Thanks a lot for pushing me in the right direction! It's working like a charm. Will fix this also for the QUnit warning in a follow up PR: https://github.com/rwjblue/ember-cli-content-security-policy/blob/f985629287f3466b02fcd94edda84f0b493595cc/index.js#L236 https://github.com/rwjblue/ember-cli-content-security-policy/blob/f985629287f3466b02fcd94edda84f0b493595cc/node-tests/e2e/test-support-test.js#L195